src/wallet/coinselection.cpp

```diff
@@ -444,6 +444,12 @@ void SelectionResult::AddInput(const OutputGroup& group)
     m_use_effective = !group.m_subtract_fee_outputs;
 }
 
+void SelectionResult::AddInputs(const std::set<COutput>& inputs, bool m_subtract_fee_outputs)
```

Comment:
In e6b9e23699a5dc8e08711004f51c877fa3912db1 "wallet: encapsulate pre-selected-inputs lookup into its own function"

nit: only member variables should have the `m_` prefix.

Reply:
yeah, removing it..